theano/gof/op.py

```diff
@@ -733,13 +739,17 @@ def R_op(self, inputs, eval_points):
                                   wrt=inputs,
                                   eval_points=eval_points)
 
+        Notes
+        -----
+        It's possible to do Rop with only Lop implemented.
+        By default, Rop performs twice reverse mode grad (Lop) to
+        obtain forward grad (Rop).
+
         """
-        raise NotImplementedError(
-            "%s of class %s does not "
-            "implement R_op. If this is a theano op, write to the "
-            "theano-dev mailing list for assistance. If it is your "
-            "own op, implement the R_op method." %
-            (self, self.__class__.__name__))
+        outputs = self.make_node(inputs).outputs
+        lop_inputs = [o.type() for o in outputs]
+        lop_outputs = self.L_op(inputs, outputs, lop_inputs)
+        return theano.Lop(lop_outputs, lop_inputs, eval_points)
```

Comment:
This would need some tests to be sure we don't break it.

It could be a simple test that create a new op with Lop implemented, but not Rop and make sure we can take the Rop. This new Op could inherit from an existing op and delete the Rop method.